oldScripts/pl_to_spreadsheet.py

```diff
@@ -0,0 +1,102 @@
+# Author: Naveen Nathan
```

Comment:
We should be asynchronous programming to handle get and post requests if possible
https://docs.python.org/3/library/asyncio.html. If the API request from PraireLearn takes 10 seconds to fetch data from PraireLearn, the thread is in a blocking state and is wasting resources when the thread can be picking up another task while waiting for the server to respond. 

Asynchronous programming would allow you to switch a thread onto another process and increase efficiency for the use of threading.

Reply:
Good idea, Manan! As of now, there is no logic to be executed concurrently, because this PR is just for retrieving data. However, this is a suggestion we can keep this in mind as the project progresses.